OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -270,7 +270,7 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 								Name = botType,
 								Bot = botType,
 								Slot = parts[0],
-								Country = "random",
+								Race = "Random",
```

Comment:
I'm going to nitpick once more, could we keep this as `random` please? For consistency.